### PR TITLE
Fix bot.players

### PR DESCRIPTION
**Pull request by rom1504**
This is only the first commit of the fixing of bot.players, maybe don't merge it yet.

What changed : 
* players now have a UUID
* packets often don't give the username anymore, only the UUID

bot.players was previously indexed by the player name, it should now probably be indexed by the UUID. A uuidToUsername function and a usernameToUuid function might be useful.
This means all mention of bot.players need to be changed throughout the code.

### CODE REVIEW

**Comment by rom1504** [comment]
In the end I let bot.players be indexed by the username and added a bot.uuidToUsername, that's enough.

Chatterbox.js now works :)

Ready to merge

File: examples/chatterbox.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ bot.on('login', function() {
   console.log("settings", bot.settings);
 });
 bot.on('time', function() {
-  console.log("current time", bot.time);
+  //console.log("current time", bot.time);
```
Comment (roblabla) [inline examples/chatterbox.js]: Could you uncomment this ? I have a feeling this isn't intentional :P

Comment (rom1504) [inline examples/chatterbox.js]: Actually this is intentional, this display one line every ~200ms in the console, making everything else the chatterbox example display unreadable. You can see several other things are commented in that example for the same reason ;) (not by me)

Comment (roblabla) [inline examples/chatterbox.js]: oh ok.

**Pull request merged**